DevCGlobalDirectory.md

```diff
@@ -365,6 +365,19 @@
 </details>
 
 
+<details>
+  <summary>
+    <b>DevC Nairobi</b>
+  </summary>
+
+  - Lead(s):
+    - [Eddie Kago](https://github.com/scripttie)
+    - [Carol Kariuki]
+  - Facebook Group: [https://www.facebook.com/groups/devcnairobi]
+  - Github: [Link to local DevC GitHub repo (if any)]
```

Comment:
**suggestion:** @TabithaKavyu I think you should leave out the link, since IIRC devc Nairobi doesn't really have a GH repo. Wdyt?

Reply:
Which link in this case? Her Github link or DevCNairobi's

Reply:
```suggestion

```

Leave out the link all together since AFAICT we don't have any.. Hope you can read diffs